app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,39 @@
+class RestaurantsController < ApplicationController
+  
+  def index
+    @restaurant = Restaurant.all
+  end
+
+  def show
+    @restaurant = Restaurant.find(params[:id])
+  end
+
+  def new
+    @restaurant = Restaurant.new
+  end
+
+  def edit
+    @restaurant = Restaurant.find(params[:id])
+  end
+
+  def create
+    @restaurant = Restaurant.create(restaurant_params)
+    redirect_to root_path 
+  end
+
+  def update
+    @restaurant = Restaurant.find(params[:id])
+    redirect_to restaurant_path(@restaurant)
```

Comment:
🔨 @Alisher778 you deleted too many lines here, you need to call `.update` to store/save the updated form data